src/neo/ProtocolSettings.cs

```diff
@@ -61,18 +63,63 @@ private ProtocolSettings(IConfigurationSection section)
                 };
             IConfigurationSection section_sl = section.GetSection("SeedList");
             if (section_sl.Exists())
-                this.SeedList = section_sl.GetChildren().Select(p => p.Get<string>()).ToArray();
+            {
+                this.SeedList = section_sl.GetChildren()
+                      .AsParallel()
+                      .Select(p => GetIpEndPoint(p.Get<string>()))
```

Comment:
If the DNS query takes 1 minute, then our program will be blocked for 1 minute. I think we should move this logic to `LocalNode` and then query it asynchronously when the node starts.